### ISSUE TITLE
Staging Synthetic Tests - Routing issues

### ISSUE THREAD

**Issue by LindseySaari** [department-of-veterans-affairs/va.gov-team#53960]
## Description

The following routes are only available behind the socks proxy:

- http://staging-platform-api.vfs.va.gov/services/appeals/supplemental_claims/v0/healthcheck
- http://staging-platform-api.vfs.va.gov/services/appeals/v2/upstream_healthcheck
- http://staging-platform-api.vfs.va.gov/higher_level_reviews/v0/upstream_healthcheck
- http://staging-platform-api.vfs.va.gov/services/appeals/notice_of_disagreements/v0/upstream_healthcheck
- http://staging-platform-api.vfs.va.gov/services/appeals/supplemental_claims/v0/upstream_healthcheck

These same routes _should_ be available behind revproxy as well (staging-api.va.gov). We should check to see if there are any kong routing changes that need to be made. We saw a similar issue for dev.

**Comment by LindseySaari** [comment]
These routes are also 404-ing and after checking, we can remove.

Returning “no route matching”:
https://staging-api.va.gov/services/fhir/v0/r4/metadata
http://staging-platform-api.vfs.va.gov/services/fhir/v0/r4/metadata
http://staging-platform-api.vfs.va.gov/services/fhir/v0/r4/openapi.json
https://staging-api.va.gov/services/fhir/v0/r4/openapi.json
http://staging-platform-api.vfs.va.gov/services/fhir/v0/dstu2/metadata
https://staging-api.va.gov/services/fhir/v0/dstu2/metadata
http://staging-platform-api.vfs.va.gov/services/fhir/v0/argonaut/data-query/openapi.json
https://staging-api.va.gov/services/fhir/v0/argonaut/data-query/openapi.json
http://staging-platform-api.vfs.va.gov/higher_level_reviews/v0/healthcheck
https://staging-api.va.gov/higher_level_reviews/v0/healthcheck
https://staging-api.va.gov/services/address_validation/healthcheck -only rev proxy route does not work!
https://staging-api.va.gov/services/fhir/v0/dstu2/openapi.json
http://staging-platform-api.vfs.va.gov/services/fhir/v0/dstu2/openapi.json
https://staging-api.va.gov/services/fhir/v0/argonaut/data-query/openapi.json
http://staging-platform-api.vfs.va.gov/services/fhir/v0/argonaut/data-query/openapi.json

**Comment by laineymajor** [comment]
slack thread about the routes - https://dsva.slack.com/archives/C04M7T1CZAS/p1677010251307779

Sounds like the only route not accounted for is http://staging-platform-api.vfs.va.gov/services/address_validation/healthcheck which I think may be a Kong routing issue because https://staging-api.va.gov/services/address_validation/healthcheck works just fine

**Comment by LindseySaari** [comment]
Additionally, [I reached out about](https://dsva.slack.com/archives/C03UA9MV1EH/p1677081396625429?thread_ts=1671458801.290079&cid=C03UA9MV1EH) the routes ONLY available behind SOCKS. We had a similar issue in dev.

**Comment by LindseySaari** [comment]
Still waiting on a response about the SOCKS only endpoints. I may remove them from synthetics if we don't hear anything

**Comment by laineymajor** [comment]
@LindseySaari Any update on the SOCKS only endpoints?

**Comment by LindseySaari** [comment]
Still [no reply](https://dsva.slack.com/archives/C03UA9MV1EH/p1677081396625429?thread_ts=1671458801.290079&cid=C03UA9MV1EH) on the SOCKS only endpoints

**Comment by laineymajor** [comment]
Seems like this is not a priority for folks. 
Will remove this from the synthetic tests after announcing in relevant slack threads.

**Comment by LindseySaari** [comment]
A [PR has been opened to remove the failing synthetics. ](https://github.com/department-of-veterans-affairs/devops/pull/12609). It appears that the revproxy based routes **are not** available currently, so this PR comments out the irrelevant endpoints for synthetic testing. [The team has noted that they will reach out](https://dsva.slack.com/archives/C03UA9MV1EH/p1677522944248709?thread_ts=1671458801.290079&cid=C03UA9MV1EH) when these endpoints have been released and we can restore the tests.

**Comment by LindseySaari** [comment]
Terrraform has been applied ✅
